sympy/integrals/tests/test_transforms.py

```diff
@@ -707,6 +707,10 @@ def test_laplace_transform():
     assert inverse_laplace_transform(
         f(w), w, t, plane=0) == InverseLaplaceTransform(f(w), w, t, 0)
     assert LT(f(t)*g(t), t, s) == LaplaceTransform(f(t)*g(t), t, s)
+    # Issue #24294
+    assert LT(b*f(a*t), t, s) == b*LaplaceTransform(f(t), t, s/a)/a
```

Comment:
I'm not quite sure how to interpret `LaplaceTransform(f(t), t, s/a)`. Do I understand correctly that `LaplaceTransform(f(t), t, s/a)` is `F(s/a)` where `F(s)` is `LaplaceTransform(f(t), t, s)`? In other words:
```python
In [11]: LaplaceTransform(f(t), t, s).subs({s:s/a})
Out[11]: 
                ⎛         s⎞
LaplaceTransform⎜f(t), t, ─⎟
                ⎝         a⎠
```
Is that equivalent to replacing `s` with `s/a` in the integral definition of the transform? Or I guess it isn't which is why the whole thing is divided by `a`...

Reply:
> Is that equivalent to replacing `s` with `s/a` in the integral definition of the transform?

Yes, it is.  If you have a Laplace transform pair `x(t)` and `X(s)`, then the Laplace transform of `x(a*t)` is `X(s/a)/a` for positive `a`.